app/ide-desktop/lib/dashboard/src/components/Loader.tsx

```diff
@@ -2,28 +2,76 @@
  * @file
  * A full-screen loading spinner.
  */
-import * as tw from 'tailwind-merge'
+import * as twv from 'tailwind-variants'
```

Comment:
nit: i'd probably `twVariants` over `twv`. or `tv` i guess, to match the import we are using